### ISSUE TITLE
NSP Security Vulnerability - Update to deep-extend@0.5.1

### ISSUE THREAD

**Issue by shellscape** [dominictarr/rc#111]
NSP is reporting a vulnerability for `Prototype Pollution` on a direct dependency of this module:

```console
┌────────────┬────────────────────────────────────────────────────────────────────┐
│            │ Prototype Pollution                                                │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Name       │ deep-extend                                                        │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ CVSS       │ 2 (Low)                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Installed  │ 0.4.2                                                              │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Vulnerable │ <=0.5.0                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Patched    │ >=0.5.1                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Path       │ webpack-command@0.1.0 > update-notifier@2.5.0 >                    │
│            │ latest-version@3.1.0 > package-json@4.0.1 > registry-url@3.1.0 >   │
│            │ rc@1.2.6 > deep-extend@0.4.2                                       │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ More Info  │ https://nodesecurity.io/advisories/612                             │
└────────────┴────────────────────────────────────────────────────────────────────┘
```

`deep-extend@0.5.1` was released to address this, and the module should be updated to depend on that version.

**Comment by billyjanitsch** [comment]
Looks like this was done in b63377974f60bc5207c15bc8f465e28d2c7e1945, which was released as rc@1.2.7 last week.

A fresh `npm i -D webpack-command` yields a transitive dependency on rc@1.2.7 and I don't get the vulnerability warning from `npm audit`, so it looks like this is taken care of.

**Comment by rob-orr** [comment]
`deep-extend` released 0.6.0 yesterday, and this should fix this issue: https://github.com/unclechu/node-deep-extend/issues/41#issuecomment-391090870.  Anybody know when `rc` will be updated?

**Comment by dominictarr** [comment]
fixed in 1.2.8
